src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,8 @@
+import style from "./Header.module.css"
+export const Header=()=>{
+    return (
+        <header className={style.header}>
```

Comment:
I don't really understand this code. why do you need the style.primaryHeading and not just call the className primaryHeading, it will still link to the css like that. won't it??

Reply:
That is because I used CSS module for this project. It is private and not global, which helps to avoid messing up class names.  When I imported my CSS file, I named it "style" ( line 1  import "style"), so I had to name my className using {style.actualName}.  If you just use className=" ", then this will be global and not private.
I hope it clears up a bit. If you have more questions, ask me.